Formula/ghostscript.rb

```diff
@@ -30,43 +29,55 @@ class Ghostscript < Formula
   end
 
   depends_on "pkg-config" => :build
+  depends_on "fontconfig"
+  depends_on "freetype"
+  depends_on "jbig2dec"
+  depends_on "jpeg"
+  depends_on "libidn"
+  depends_on "libpng"
   depends_on "libtiff"
+  depends_on "little-cms2"
+  depends_on "openjpeg"
+
+  uses_from_macos "expat"
+  uses_from_macos "zlib"
 
   on_macos do
     patch :DATA # Uncomment macOS-specific make vars
   end
 
-  on_linux do
-    depends_on "fontconfig"
-    depends_on "libidn"
-  end
-
   # https://sourceforge.net/projects/gs-fonts/
   resource "fonts" do
     url "https://downloads.sourceforge.net/project/gs-fonts/gs-fonts/8.11%20%28base%2035%2C%20GPL%29/ghostscript-fonts-std-8.11.tar.gz"
     sha256 "0eb6f356119f2e49b2563210852e17f57f9dcc5755f350a69a46a0d641a0c401"
   end
 
-  # Fix build on ARM Big Sur, updating config.{guess,sub}
-  # https://bugs.ghostscript.com/show_bug.cgi?id=703095
-  patch do
-    url "https://raw.githubusercontent.com/Homebrew/formula-patches/98c39e09/ghostscript/config.patch"
-    sha256 "155cf2ee5a498d441c3194ba3d75cb7812beaa3f507a72017174a884bf742862"
-  end
-
   def install
-    on_linux do
-      # Fixes: ./soobj/dxmainc.o: file not recognized: File truncated
-      ENV.deparallelize
-    end
+    # Fix vendored tesseract build error: 'cstring' file not found
+    # Remove when possible to link to system tesseract
+    ENV.append_to_cflags "-stdlib=libc++" if ENV.compiler == :clang
+
+    # Fix VERSION file incorrectly included as C++20 <version> header
+    # Remove when possible to link to system tesseract
+    rm "tesseract/VERSION"
```

Comment:
Is `tesseract` used for OCR? If so, it's likely someone's using that. If the package size difference isn't in the order of 100s of MBs I'd say we keep it.

Reply:
Sure. At least for now, size is difficult to compare since we are shedding other vendored dependencies at same time.

The other thing I saw in ArchLinux was a `libgpdl` linkage removal:
```
  # http://git.ghostscript.com/?p=ghostpdl.git;a=commit;h=40dc5b409c6262b18b4bf5386b5482ead4c511e3
  # libs link unwanted to libgpdl that isn't installed
  rm -rf gpdl
```
I haven't had the chance to see if this conflicts with anything in Homebrew.